includes/admin/class-wcmp-export.php

```diff
@@ -334,9 +337,18 @@ public function add_shipments(array $order_ids, bool $process)
 
             $extraOptions = WCX_Order::get_meta($order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
             $colloAmount  = $extraOptions["collo_amount"] ?? 1;
+            $isMailboxPackage  = 'mailbox' === $this->getPackageTypeFromOrder($order);
+            $isMultiColloAllowed = (
```

Comment:
Ik vind dit nicer en makkelijker te lezen: `in_array($order->get_shipping_country(), [self::COUNTRY_CODE_NL, self::COUNTRY_CODE_BE])`

Reply:
Deze variabele kan misschien beter `$isMultiColloCountry` heten, hij impliceert voor mij nu dat hij meerdere dingen zou valideren, waardoor de "is bbp" check hierna een beetje gek wordt